README.md

```diff
@@ -82,7 +96,21 @@ client info to try the demo).
 
 ## Setup
 
-AppAuth supports four options for dependency management.
+AppAuth supports four options for dependency management. For **tvOS**, please use the 'TV' subspec in CocoaPods and `AppAuthTV` when using Swift Package Manager/Carthage.
```

Comment:
I think the tvOS stuff should be under each respective package manager's section.

Reply:
+1, and it looks like it's covered in the version I see below